script/docker-startup

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -f tmp/pids/server.pid
+
```

Comment:
I remember this being given as an example of the kind of thing that gets put in an `ENTRYPOINT` script in the [Docker Rails book. ](https://pragprog.com/titles/ridocker/docker-for-rails-developers/) (It's on O'Reilly. The chapter is "Rails tmp/pids/server.pid Not Cleaned Up") 

That said it's straightforward enough that leaving it has part of the command seems fine to me.